### PR TITLE
TW-594 Collections and workspaces on private API network

### PR DESCRIPTION
**Pull request by claireriley**
Collections and Workspaces on Private API Network for v10 release

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 1]
```diff
@@ -30,85 +30,73 @@ contextual_links:
...
+In the Private API Network, you can filter elements by name using the search box. Select **Added by** to filter based on the person who added the element. Select **Type** to filter by the type of element. Select **Sort by** to sort elements based on name and date added. You can also filter folders, sub-folders, and elements based on name and date added.
 
-<img alt="Private API List" src="https://assets.postman.com/postman-docs/private-api-network-list-v9.jpg"/>
+<img alt="Private API List" src="https://assets.postman.com/postman-docs/v10/private-api-network-list-v10.jpg"/>
 
-To review information about an API, select it from the list. You can view the API description, the editors who have worked on the API, and the workspace in which it exists.
+To review information about an element, select it from the list. You can view the element's description and the editors who have worked on it. For workspaces, you can view all of the collections and APIs that reside in them. For collections, you can view available documentation. For APIs, you can view definitions and associated collections.
```
Comment (HeatherStenson) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:76]:
```suggestion
To review information about an element, select it from the list. You can view the element's description and the editors who have worked on it. For workspaces, you can view all of the collections and APIs inside them. For collections, you can view available documentation. For APIs, you can view definitions and associated collections.
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 2]
```diff
@@ -126,97 +114,152 @@ Postman will send an email to new API Network Managers about their updated role.
...
+To add a workspace to the Private API Network from the API Builder:
 
-## Adding your APIs
+1. Open the workspace you want to add to the Private API Network.
+1. In the workspace overview, select **Add to API Network**.
+1. (Optional) Select a folder or create one to keep elements organized.
+1. Select **Add**.
 
-To add an API to the Private API Network, it must be in the [API Builder](/docs/designing-and-developing-your-api/creating-an-api/) in a team or public workspace. You can't add an API to the Private API Network unless all team members have at least view access to the API. Learn more about team [roles and permissions](/docs/collaborating-in-postman/roles-and-permissions/).
+You can also add a workspace while in your team's Private API Network:
```
Comment (HeatherStenson) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:204]: Maybe: 
```suggestion
You can also add a workspace from inside your team's Private API Network:
```

❓

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 3]
```diff
@@ -126,97 +114,152 @@ Postman will send an email to new API Network Managers about their updated role.
...
 
-* **If your team uses the [optional approval process](#using-the-approval-process-workflow)**, users with an Editor role must [submit a request](#editor-requesting-to-add-an-api) to add an API to the Private API Network and an API Network Manager must [approve the request](#api-network-manager-reviewing-requests-to-add-apis). API Network Managers can add APIs directly to the Private API Network following the instructions here.
-* **If your team doesn't use the optional approval process**, any user with an Editor role for an API can add it to the Private API Network following the instructions here.
+1. Open your [Private API Network](https://go.postman.co/network/private).
+1. Select **Add to network**
```
Comment (HeatherStenson) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:207]:
```suggestion
1. Select **Add to network**.
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 4]
```diff
@@ -30,85 +30,73 @@ contextual_links:
...
-    name: "Next Steps"
-  - type: link
-    name: "Documenting your API"
-    url: "/docs/publishing-your-api/documenting-your-api/"
-  - type: link
-    name: "Generating collections from the API"
-    url: "/docs/designing-and-developing-your-api/the-api-workflow/"
     search_keyword: "Private API Network, API sharing, folders, network listing, filtering apis, publish versions, github import, private apis, adding apis"
 
 ---
```
Comment (HeatherStenson) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:36]: IIRC, this feature is Enterprise-only going forward? If so we should probably have a note to that effect at the top of the page.

**Review by HeatherStenson — APPROVED** [review]
Looks great! 🍊 🌟

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 5]
```diff
@@ -225,75 +268,69 @@ All API versions are automatically visible to API Editors and Viewers within a P
...
 You can also use **Create folder** to create sub-folders inside a folder.
 
-![New folder in Private Network](https://assets.postman.com/postman-docs/add-new-folder-api-network-overview-v9.jpg)
+Use **Search elements and folders** to search across folders, sub-folders, and elements in your Private API Network.
 
-You can use **Search APIs and folders** to search across folders, sub-folders, and APIs in your Private API Network.
+### Editing element listings
 
-### Editing API listings
+Select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the element you would like to edit from the network, then select **Edit element** to change its location. Select **Edit** to save your changes.
```
Comment (postmanjohn) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:310]: We should also add that for collections **Edit element** allows you to edit the summary, and associated environments.

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 6]
```diff
@@ -126,97 +114,152 @@ Postman will send an email to new API Network Managers about their updated role.
...
+### Adding APIs
 
-The APIs that you add to the Private API Network reflect the latest state of the API in your team workspace. In other words, changes made to the API in the workspace are reflected in the network in real time. Additionally, all API versions that are visible to consumers are available on the Private API Network. Read more about [publishing specific API versions](#publishing-specific-api-versions).
+If your team uses the [optional approval process](#using-the-approval-process-workflow), only an API Network Manager can add APIs directly to the Private API Network. API Editors must [request to add an API](#requesting-to-add-an-api). If your team doesn't use the approval process, any user with Editor access for the API can add it to the Private API Network.
 
-### Publishing specific API versions
+To add an API to the Private API Network from the API Builder:
+
+1. Open the API you want to add to the Private API Network.
+1. In the API overview, select **Add to API Network**.
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:246]: I think this changed to be just Publish API, but verify.

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 7]
```diff
@@ -225,75 +268,69 @@ All API versions are automatically visible to API Editors and Viewers within a P
 1. Select **Consumers can view this version**.
 1. Select **Save Changes**.
 
-### Importing APIs from GitHub
+#### Importing APIs from code repositories
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:271]: We have competing edits between this and TW-493, with the latter one being correct.

**Review by jkonrath-postman — APPROVED** [review]
I'll approve, but double-check the flow from API Builder, and there are some competing edits between this and TW-493 that need to be sorted out.

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 8]
```diff
@@ -126,97 +116,153 @@ Postman will send an email to new API Network Managers about their updated role.
...
+
+1. Open your [Private API Network](https://go.postman.co/network/private).
+1. Select **Add to network**
+1. Search for and select the API you want to add.
+1. (Optional) Select a folder or create one to keep elements organized.
+1. Select **Add**.
+
+When you add an API to the [Private API Network](https://go.postman.co/network/private), it's visible to your Postman team.
+
+The APIs that you add to the Private API Network reflect the latest state of the API in your team workspace. In other words, changes made to the API in the workspace are reflected in the network in real time.
```
Comment (postmanjohn) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:263]: we should probably denote that published changes are reflected, not just changes alone.

**Comment by postmanjohn** [comment]
Looks good other than the one comment I left :+1:

**Pull request merged**